packages/beacon-node/src/network/gossip/encoding.ts

```diff
@@ -96,9 +97,9 @@ export class DataTransformSnappy {
    * Takes the data to be published (a topic and associated data) transforms the data. The
    * transformed data will then be used to create a `RawGossipsubMessage` to be sent to peers.
    */
-  outboundTransform(topicStr: string, data: Uint8Array): Uint8Array {
+  outboundTransform(_topicStr: string, data: Uint8Array): Uint8Array {
     if (data.length > this.maxSizePerMessage) {
-      throw Error(`ssz_snappy encoded data length ${length} > ${this.maxSizePerMessage}`);
+      throw Error(`ssz_snappy encoded data length ${data.length} > ${this.maxSizePerMessage}`);
```

Comment:
[window.length](https://developer.mozilla.org/en-US/docs/Web/API/Window/length) was used instead.

Reply:
how was the code was compiled?

Reply:
I think this is because we include [dom](https://github.com/ChainSafe/lodestar/blob/f6d3bce80297023aaf890f201f5e94f1e701e46c/tsconfig.build.json#L4) lib in our `tsconfig.build.json`.